guides/doc-Provisioning_Guide/topics/Cloud-GCE.adoc

```diff
@@ -112,3 +166,25 @@ ifeval::["{build}" == "foreman"]
 . If you use the Katello plugin, click the *Parameters* tab and ensure that a parameter exists that provides an activation key. If not, add an activation key.
 endif::[]
 . Click *Submit* to save the host entry.
+
+.For CLI Users
+
+To create a host, enter the following command:
+
+[options="nowrap" subs="+quotes"]
+----
+# hammer host create --name "_GCE_VM_" \
+--organization "_Your_Organization_" \
+--location "_Your_Location_" \
+--compute-resource _gce_cr_name_
+--compute-profile "_gce_profile_name_" \
+--image _gce_image_name_
+--provision-method 'image' \
+--root-password _changeme_ \
```

Comment:
lol, let's change this to something more generic and in line with other examples of creating a host.